website/content/en/blog/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+date: 2022-10-10
+linkTitle: "Kubeflow on AWS Blog"
+description: "Welcome to the **blog** section of Kubeflow on AWS!"
+type: "docs"
+layout: "single"
+---
+
+In this section, we share posts, articles, and videos illustrating how some of our customers have leveraged **AWS distribution of Kubeflow** to optimize large-scale and compute-intensive machine learning workloads, we provide ML infrastructure recommendations and best practices, and we inform you on Kubelow on AWS coming training sessions and available recordings.
+
+Want to **try the latest version of Kubeflow on AWS**? Follow the steps in [Kubeflow on AWS documentation](http://localhost:1313/kubeflow-manifests/main/docs/deployment/prerequisites/).
+
+[**Read**](#read) from our curated list of posts or [**Watch**](#watch) a recorded session.
+
+## Read
+
+A curated list of blogs illustrating how to build flexible, scalable ML workflows on Kubernetes using AWS distribution of Kubeflow. 
+Find case studies, use cases, best practices, benchmarks, and more.
+
+1. **[ Architecture ]** 
```

Comment:
what do you think about avoiding classify the blog posts and recordings? like[Architecture or case study] It will be a difficult to maintain and becomes difficult if blog touches multiple areas

We can instead add it in 1-2 sentences as needed

Reply:
I just removed everything for this first release.